api/src/Entity/Camp.php

```diff
@@ -116,7 +138,7 @@ class Camp extends BaseEntity implements BelongsToCampInterface {
     #[InputFilter\CleanHTML]
     #[Assert\NotBlank]
     #[ApiProperty(example: 'SoLa 2022')]
-    #[Groups(['Default', 'camp:update'])]
+    #[Groups(['read', 'create'])]
```

Comment:
```suggestion
    #[Groups(['read', 'write'])]
```
Sonst kann man den Campnamen nie mehr updaten